R/add_missing_events.R

```diff
@@ -0,0 +1,325 @@
+#Author: Paigan Aspinall
```

Comment:
Bit confused by this file. It's the weekly report but in the add_missing_events function?